### PR TITLE
up one php to 8.1

### PR DESCRIPTION
**Pull request by lizhichao**

### CODE REVIEW

**Comment by joanhey** [comment]
Please update to PHP 8.2.

**Comment by joanhey** [comment]
#7788 coauthored

**Comment by joanhey** [comment]
@lizhichao
One pass the tests with php 8.1 and 8.2, but with a lot of deprecation warnings.
With php 8.1
https://github.com/TechEmpower/FrameworkBenchmarks/actions/runs/3672015242/jobs/6207789060#step:9:487

With php 8.2, more warnings: 
https://github.com/TechEmpower/FrameworkBenchmarks/actions/runs/3678986053/jobs/6222906454#step:9:493

File: frameworks/PHP/one/composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   ],
   "require": {
     "php": ">=7.1.0",
-    "lizhichao/one": ">=2.0.1",
+    "lizhichao/one": "dev-master",
```
Comment (joanhey) [inline frameworks/PHP/one/composer.json:20]: Please create a new version, and use it here.

**Review by joanhey — CHANGES_REQUESTED** [review]

File: frameworks/PHP/one/composer.json (hunk 1 above)
Comment (lizhichao) [reply frameworks/PHP/one/composer.json]: done

**Pull request merged**